src/libstd/os/raw/long.md

```diff
@@ -0,0 +1,8 @@
+Equivalent to C's `signed long` (`long`) type.
+
+This type will usually be [`i64`], but is sometimes [`i32`] \(i.e. [`isize`]\) on 32-bit systems. Technically, the standard only requires that it be at least 32 bits, or at least the size of an [`int`].
```

Comment:
Also note that it is equivalent to `i32` even on 64bit Windows.

Reply:
I was not aware of this; I'll remove the remarks about `isize` and `usize` in these docs.